app/assets/stylesheets/application.scss

```diff
@@ -14,17 +14,35 @@
  *= require_self
  */
 
- @import "bulma";
- @import "font-awesome";
+$text: hsl(270, 50%, 29%);
+$text-light: hsl(270, 50%, 48%);
+$text-strong: hsl(270, 50%, 21%);
```

Comment:
@ka8725, to get the color as in the picture we need to add `$title-color: hsl(270, 50%, 40%);` here
![image](https://github.com/widefix/pocketmoney/assets/81028629/928cf3d1-fdcd-4d40-bd6e-0da4f571b106)

Let's look at an option:
 - create the unique style for such title font only for the landing page